src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -15,6 +15,7 @@ namespace Microsoft.PowerShell.Commands
     /// The ConvertTo-Json command.
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-macos`
Validate Update-Help from the Web for all PowerShell modules..Validate Update-Help for module 'Microsoft.PowerShell.Core' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```
ConvertTo-Json.Should handle null
```powershell
Value cannot be null. (Parameter 'type')
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Json.Tests.ps1: line 88
88:         $null | ConvertTo-Json -Compress | Should -Be 'null'

```